third_party/tsl/tsl/cuda/stub.bzl

```diff
@@ -0,0 +1,22 @@
+def cuda_stub(name, library):
+    """Generates a CUDA stub from a list of symbols.
+
+    Generates two files:
+    * library.inc, which contains a list of symbols suitable for inclusion by
+        C++, and
+    * library.tramp.S, which contains assembly-language trampolines for each
+      symbol.
+    """
+    native.genrule(
+        name = name,
+        srcs = ["{}.symbols".format(library)],
```

Comment:
Nit: it might be slightly cleaner to pass `src` explicitly as a separate macro parameter.

Deriving the output files from the name is fine.

Reply:
Done.